cfpb/consumerfinance.gov
# Update to DS 3.4.14

Update to DS 3.4.14, which does a few things:

- Refactors the date picker to make it more consistently sized across web browsers, and nearer to the dimensions of a text input.
- Adds a non-clickable pill pattern. This was in use in the FIG, but it could be clicked in the past, but do nothing. Prod currently only shows the text "Archived", which is a bug.
- 


## Additions

-


## Removals

- Removes unused percentage width utility classes.


## Changes

- Update to DS 3.4.14.
- Fix issue where FIG tag class name needed to be updated.
- Fix issue where no-js iregs search tag class name needed to be updated.


## How to test this PR

1. Visit http://localhost:8000/about-us/blog/ and see that the date pickers are 35px high, instead of 37px, which matches the height of the drop-down menus.
- Visit http://localhost:8000/data-research/small-business-lending/filing-instructions-guide/2024-guide-archive-v2/ and see that the "Archived" text is in a non-clickable pill pattern.
- Visit http://localhost:8000/activity-log/ and see that the width of the columns is unchanged to prod.
- Visit http://localhost:8000/rules-policy/regulations/search-regulations/results/?regs=1002&q=mortgage and turn off JS and see that the filter tag's "X" icon disappears.
